allensdk/core/json_utilities.py

```diff
@@ -164,7 +164,7 @@ def json_handler(obj):
         return float(obj)
     elif isinstance(obj, np.integer):
         return int(obj)
-    elif (isinstance(obj, np.bool) or
+    elif (isinstance(obj, bool) or
           isinstance(obj, np.bool_)):
```

Comment:
Wanted to make sure this one shouldn't also be changed to `bool_` as `interval_unionize/data_utilities.py` was switched from `np.bool_` to `bool_`.